pkg/reconciler/ip.go

```diff
@@ -0,0 +1,46 @@
+package reconciler
+
+import (
+	"context"
+	"time"
+
+	"github.com/k8snetworkplumbingwg/whereabouts/pkg/logging"
+)
+
+const (
+	defaultReconcilerTimeout = 30
+)
+
+func InvokeIPReconciler(returnErr chan error) {
```

Comment:
I would use another function name - `ReconcileIPs` maybe ?

Reply:
Sure let's go with that